release/rllib_tests/learning_tests/yaml_files/ppo/new_stack/ppo_breakout.py

```diff
@@ -0,0 +1,82 @@
+import gymnasium as gym
+
+from ray.rllib.algorithms.ppo import PPOConfig
+from ray.rllib.connectors.env_to_module.frame_stacking import FrameStackingEnvToModule
+from ray.rllib.connectors.learner.frame_stacking import FrameStackingLearner
+from ray.rllib.env.single_agent_env_runner import SingleAgentEnvRunner
+from ray.rllib.env.wrappers.atari_wrappers import wrap_atari_for_new_api_stack
+from ray import tune
+
+
+def _make_env_to_module_connector(env):
+    return FrameStackingEnvToModule(num_frames=4)
+
+
+def _make_learner_connector(input_observation_space, input_action_space):
+    return FrameStackingLearner(num_frames=4)
+
+
+# Create a custom Atari setup (w/o the usual RLlib-hard-coded framestacking in it).
+# We would like our frame stacking connector to do this job.
+def _env_creator(cfg):
+    return wrap_atari_for_new_api_stack(
+        gym.make("ALE/Breakout-v5", **cfg, **{"render_mode": "rgb_array"}),
+        # Perform through ConnectorV2 API.
+        framestack=None,
```

Comment:
Is this intended bheavior or only for test? Should users keep using framestack wrappers or use conectors?

Reply:
Thanks for the clarification. We should later collect all possible speedups we find and remember together and give users a speed-up guide that navigates them a bit through the  labyrinth - especially in the areas where these changes would contradict our design mantras. 

On the side: why only after sampling and before training? does framestack not usually require a different encoder?